### DISCUSSION TITLE
Stuck On Setting Up Your Codespace

### DISCUSSION THREAD

**Discussion by SoloLeveling7seven** [community/community#58374] [Codespaces] [answered]
### Select Topic Area

Question

### Body

How can i solve this problem ,i am not able to open codespace in browser , it was working fine earlier but after i cleared my browsing history and logged in again its getting stuck on setting up your codespace 

![image](https://github.com/community/community/assets/109235836/68bf4252-0539-4b4c-8ad2-1011c923c1b4)

The Codespace is Active But When i click on it it shows this..!!

![image](https://github.com/community/community/assets/109235836/fed1f318-1e32-4a59-824e-6c1ac5c0c2c7)

**Comment by ariG23498** [comment]
I am facing a similar issue -- Used chrome and safari to check -- both giving the same results.

**Reply by ariG23498** [reply]
Just as an update -- The problem still persists. CC: @dmgardiner25

**Comment by dmgardiner25** [comment]
@SoloLeveling7seven It looks like you have been able to successfully connect to your codespace after reporting this. Could you confirm if you are still seeing this issue?

**Reply by DineshhChoudhary** [reply]
@dmgardiner25 I am still facing this issue from last 2 days.

**Reply by SoloLeveling7seven** [reply]
I connected to vpn and opend codespace and it works and now i dont need to connect to vpn again ,its work noramlly now

**Reply by archakkhandayit** [reply]
Connect touch vpn extention and open codespace for the first time . Then don't ever need to connect vpn

**Comment by dineshh-choudhary** [comment]
I am also facing the same issue from last 2 days. The screen is getting stuck at "Setting up your codespace"

**Comment by SoloLeveling7seven** [comment, marked as answer]
i found solution for this just add touch vpn extension in your chrome and connect to vpn and open codespace once its open you can disconnect vpn and no need to connect vpn ever again

**Reply by bttow** [reply]
@SoloLeveling7seven I was facing the same problem. I did what you advised - downloaded the Touch VPN extension for my browser and connected to it. Then opened codespace again and voila! It worked!

Thank you.

**Reply by mrxsierra** [reply]
it worked for me too...

**Reply by GojuNoah** [reply]
Not working for me. Trying to access the codespace from a work controlled computer. still getting white screen with "setting up your codespace"

**Reply by Sudarshanhalse1** [reply]
> i found solution for this just add touch vpn extension in your chrome and connect to vpn and open codespace once its open you can disconnect vpn and no need to connect vpn ever aga


> i found solution for this just add touch vpn extension in your chrome and connect to vpn and open codespace once its open you can disconnect vpn and no need to connect vpn ever again

> i found solution for this just add touch vpn extension in your chrome and connect to vpn and open codespace once its open you can disconnect vpn and no need to connect vpn ever again

I am trying from last 2 days, and thanks it worked by using vpn

**Reply by ecouf** [reply]
Thank you! I also tried and it worked for me.

**Reply by Pavel-Bakunovich** [reply]
Same issue here. VPN doesn't help.
What I found out is that Codespaces don't seem to work in Safari - lot's of errors in console.
It works perfectly in Chrome.

Any tricks to make it work in Safari? Does Github even claim Safari support?

**Reply by Denoroy737** [reply]
> i found solution for this just add touch vpn extension in your chrome and connect to vpn and open codespace once its open you can disconnect vpn and no need to connect vpn ever again

yeah facing same issue but vpn method worked for me. for your knowledge i am using brave browser in windows

**Reply by soft-knit** [reply]
VPN Method worked for me as I am using Brave Browser and I used Urban VPN Proxy

**Reply by abhi-4567** [reply]
but now when i add live server extension to the codspace it not loading the browser till it connected to vpn what to do

**Comment by ghost** [comment]
i am facing this issue with me to

**Comment by rajeshanupoju208** [comment]
hi Team,

i am facing adding codespace some one help me here 
vpn is not working for me

**Comment by dnjebdebhdbhdbhebh** [comment]
same

**Comment by dnjebdebhdbhdbhebh** [comment]
can some one help me??
